<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>盾构载荷预测</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	font-family:微软雅黑;
	font-size:17px;
	color: #1d649c;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>691</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QWidget" name="slurry_page">
     <widget class="QWidget" name="slurry_widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>461</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="slurry_table">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>651</width>
         <height>331</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>环号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>埋深</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>刀盘直径</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>天然抗压强度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>岩石剪切强度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>刀盘转速</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>密封舱压力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>掘进速度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>贯入度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>推力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>扭矩</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="calslurry_btn">
          <property name="text">
           <string>计算</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clearslurry_btn">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveslurry_btn">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="drawslurry_btn">
          <property name="text">
           <string>绘图</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>651</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="importslurry_btn">
          <property name="text">
           <string>导入泥水盾构数据</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>泥水盾构</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addslurry_btn">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delslurry_btn">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="earth_page">
     <widget class="QWidget" name="earth_widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>461</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="earth_table">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>651</width>
         <height>331</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>环号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>埋深</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>刀盘直径</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>掘进速率</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>刀盘转速</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>土仓压力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>弹性模量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>泊松比</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>密度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>粘聚力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>静止土压力系数</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>内摩擦角</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>岩土承载力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>推力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>扭矩</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="calearth_btn">
          <property name="text">
           <string>计算</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clearearth_btn">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveearth_btn">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="drawearth_btn">
          <property name="text">
           <string>绘图</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>651</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="importearth_btn">
          <property name="text">
           <string>导入土压盾构数据</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>土压盾构</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addearth_btn">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delearth_btn">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="slurry_btn">
       <property name="text">
        <string>泥水盾构</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="slurryearth_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="earth_btn">
       <property name="text">
        <string>土压盾构</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
